README.md

```diff
@@ -6,6 +6,7 @@ All the free ways you can host your dynamic (with backend) or static (frontend o
 1. [Surge](http://surge.sh/)
 1. [Netlify](https://www.netlify.com/) (especially for frontend projects with build step)
 1. [Github Pages](https://pages.github.com)
+1. [ZEIT Now](https://zeit.co/)
```

Comment:
now can be used for hosting dynamic sites as well, so move it to Dynamic Sites section.

Reply:
@itaditya You're right! I moved it to the Dynamic Sites section.